### PR TITLE
Pushwoosh Source doc

### PR DESCRIPTION
**Pull request by artkry**
Adding Pushwoosh Source integration doc

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/pushwoosh/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Pushwoosh Source
+---
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/pushwoosh/index.md:3]:
```suggestion
---
title: Pushwoosh Source
id: MW9K4HgBZz
beta: true
---
```

File: src/connections/sources/catalog/cloud-apps/pushwoosh/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+5. Copy the Write key from the Segment UI.
+6. Log in to your Pushwoosh account and select the Application you want to connect.
+7. Navigate to Settings > 3rd party integration > Segment, click the `Configure` button, and enter your write key from your Segment workspace.
+8. List the events you want to export using a `,` delimiter in the `Events` field, or leave it blank if you want to export all events, then click `Apply`.
+
+## Stream
+
+Pushwoosh uses our stream Source component to send Segment event data. It uses a server-side track method to send data to Segment. These events are then available in any destination that accepts server-side events, and are available in a schema in your data warehouse.
+
+Pushwoosh passes the user identifier as the `user_id` property by default.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/pushwoosh/index.md:29]: > Pushwoosh passes the user identifier as the `user_id` property by default.

Could you rewrite this to provide more clarity on the following: 

- is "user identifier" specific to pushwoosh?
- What happens if there is no "user identifier" present, do you send anonymous data with an anonymousId? If so, what is used for anonymousId?

File: src/connections/sources/catalog/cloud-apps/pushwoosh/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Events
+
+The table below lists events that Pushwoosh sends to Segment. These events appear as tables in your warehouse, and as regular events in other Destinations. Pushwoosh includes the `user_id` if available.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/pushwoosh/index.md:33]: > Pushwoosh includes the `user_id` if available.

If referring to the Segment `userId` please change from `user_id` to userId`. What happens when userId is not available, is the event still sent?

**Review by tcgilbert — CHANGES_REQUESTED** [review]

**Review by cmastr — APPROVED** [review]

File: src/connections/sources/catalog/cloud-apps/pushwoosh/index.md (hunk 2 above)
Comment (artkry) [reply src/connections/sources/catalog/cloud-apps/pushwoosh/index.md]: - "is "user identifier" specific to pushwoosh?" - yes, its Pushwoosh User ID (i've fixed it in the doc)
- "What happens if there is no "user identifier" present, do you send anonymous data with an anonymousId? If so, what is used for anonymousId?" - yes, Pushwoosh sent Pushwoosh HWID as the Segment anonymousId (i've noted it in the doc also)

File: src/connections/sources/catalog/cloud-apps/pushwoosh/index.md (hunk 3 above)
Comment (artkry) [reply src/connections/sources/catalog/cloud-apps/pushwoosh/index.md]: Fixed
as i answered in above conversation, If userId not available, Pushwoosh sent Pushwoosh HWID as Segment anonymousId
So, yes, the event still sent

File: src/connections/sources/catalog/cloud-apps/pushwoosh/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+6. Log in to your Pushwoosh account, and select the Application you want to connect.
+7. Navigate to **Settings > 3rd party integration > Segment**, click the `Configure` button, and enter your write key from your Segment workspace.
+8. List the events you want to export using a `,` delimiter in the `Events` field, or leave it blank if you want to export all events, then click `Apply`.
+
+## Stream
+
+Pushwoosh uses our stream Source component to send Segment event data. It uses a server-side track method to send data to Segment. These events are then available in any destination that accepts server-side events, and are available in a schema in your data warehouse.
+
+Pushwoosh passes the user identifier as the Segment `userId` property by default, which is equals to Pushwoosh User ID. 
+In other case, if Pushwoosh does not have an associated `userId`, Pushwoosh HWID will be passed as the Segment `anonymousId`.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/pushwoosh/index.md:32]:
```suggestion
By default, Pushwoosh passes the Pushwoosh User ID as the Segment userId property. If there is no associated User ID, Pushwoosh will pass the HWID as the Segment anonymousId.
```

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**